source/locale/zh_CN/LC_MESSAGES/reference/generated/optuna.distributions.DiscreteUniformDistribution.po

```diff
@@ -40,30 +41,34 @@ msgid ""
 ":math:`k q + \\mathsf{low} < \\mathsf{high}`, where :math:`k` is an "
 "integer."
 msgstr ""
+"如果区间 :math:`[\\mathsf{low}, \\mathsf{high}]` 不能被步数"
+" :math:`q` 整除的话， 值 :math:`\\mathsf{high}` "
+"会被替代成 :math:`k q + \\mathsf{low} \\lt"
+" \\mathsf{high}` 的最大值，其中 :math:`k` 是整数. "
 
 #: of optuna.distributions.DiscreteUniformDistribution:14
 msgid ""
 "Lower endpoint of the range of the distribution. ``low`` is included in "
 "the range."
-msgstr ""
+msgstr "该分布的取值范围下界. 其中 ``low`` 值是包含在取值范围内的. "
 
 #: of optuna.distributions.DiscreteUniformDistribution:18
 msgid ""
 "Upper endpoint of the range of the distribution. ``high`` is included in "
 "the range."
-msgstr ""
+msgstr "该分布的取值范围上界. 其中 ``high`` 值是包含在取值范围内的. "
 
 #: of optuna.distributions.DiscreteUniformDistribution:22
 msgid "A discretization step."
-msgstr ""
+msgstr "离散化步长"
 
 #: of optuna.distributions.DiscreteUniformDistribution
 msgid "引发"
 msgstr ""
 
 #: of optuna.distributions.DiscreteUniformDistribution:24
 msgid "If ``low`` value is larger than ``high`` value."
-msgstr ""
+msgstr "如果 ``low`` 的值比 ``high`` 大的话."
 
 #: ../../source/reference/generated/optuna.distributions.DiscreteUniformDistribution.rst:12
 msgid "Methods"
```

Comment:
Missing translation

Reply:
You mean "Methods"?

Reply:
I meant the `If ``low`` value is larger than ``high`` value.`

Reply:
I cannot comment on a not-changed-line